lagunita/lms/templates/header.html

```diff
@@ -0,0 +1,7 @@
+## mako
```

Comment:
This file also exists in cme/suclass/scpd.

Go ahead and symlink it everywhere.

Reply:
erm, this actually doesn't exist, it's a new gingko thing...

Reply:
symlinked scpd copy of this file though